benchmark/opperf/utils/benchmark_utils.py

```diff
@@ -148,6 +148,11 @@ def run_op_benchmarks(ops, dtype, ctx, profiler, warmup, runs):
     for op, op_params in ops.items():
         # Prepare inputs for the operator
         inputs = prepare_op_inputs(op, op_params)
+
+        # Testing Embedding op backwards is currently unsupported
```

Comment:
Is there an issue tracking this? If yes can this be implemented?

Reply:
Good point - added issue link.